dev/github/run_tests.sh

```diff
@@ -56,4 +66,5 @@ fi
 
 # Run tests
 mvn -Duser.home=/home/jenkins test -Dmaven.main.skip -Dskip.protoc=true -Dmaven.javadoc.skip -Dlicense.skip=true \
--Dcheckstyle.skip=true -Dfindbugs.skip=true -Dsort.skip -Dsurefire.forkCount=${ALLUXIO_FORK_COUNT} ${mvn_args}
+-Dcheckstyle.skip=true -Dfindbugs.skip=true -Dsort.skip -Dsurefire.forkCount=${ALLUXIO_FORK_COUNT} \
+${mvn_args} ${mvn_test_args}
```

Comment:
```suggestion
${mvn_test_args}
```

the new `mvn_test_args` should contain the previous contents of `mvn_args` too